src/modules/31-variables/components/VariableValidation/VariableValidation.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { FormInput } from '@harness/uicore'
+import type { FormikContext } from 'formik'
+import { useStrings } from 'framework/strings'
+import AllowedValuesField from '../AllowedValues/AllowedValuesField'
+
+export enum Validation {
+  FixedValue = 'FixedValue',
+  AllowedValues = 'AllowedValues',
+  Regex = 'Regex'
+}
+
+interface VariableValidationProps {
+  formik: FormikContext<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9284062)
```suggestion
  formik: FormikContext<unknown>
```